vividus-bdd-engine/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
 
     implementation project(':vividus-util')
     implementation(group: 'org.apache.commons', name: 'commons-lang3', version: versions.commonsLang3)
+    implementation(group: 'org.apache.commons', name: 'commons-collections4', version: versions.commonsCollections4)
```

Comment:
<T> List<List<T>> com.google.common.collect.Lists.partition(List<T> list, int size)

to avoid extra dependency?